src/translations/en/referenceTypes.json

```diff
@@ -1,9 +1,9 @@
 {
-  "book": "Book",
-  "chapter": "Chapter",
-  "degree": "Degree",
-  "journalArticle": "Publication in Journal",
-  "report": "Report",
+  "Book": "Book",
+  "Chapter": "Chapter",
+  "Degree": "Degree",
+  "JournalArticle": "Publication in Journal",
+  "Report": "Report",
```

Comment:
la om til CamelCase her for å kunne slå opp på disse med verdier fra backend. innvendinger?

Reply:
PascalCase ja 😉 synes det er helt greit